kubeadm/scripts/PrepareNode.ps1

```diff
@@ -11,14 +11,20 @@ This script assists with joining a Windows node to a cluster.
 .PARAMETER KubernetesVersion
 Kubernetes version to download and use
 
+.PARAMETER ContainerRuntime
+Container that Kubernetes will use. (Docker or containerD)
+
 .EXAMPLE
-PS> .\PrepareNode.ps1 -KubernetesVersion v1.17.0
+PS> .\PrepareNode.ps1 -KubernetesVersion v1.19.1 -ContainerRuntime containerD
 
 #>
 
 Param(
     [parameter(Mandatory = $true, HelpMessage="Kubernetes version to use")]
-    [string] $KubernetesVersion
+    [string] $KubernetesVersion,
+    [parameter(HelpMessage="Container runtime that Kubernets will use")]
+    [ValidateSet("containerD", "Docker")]
+    [string] $ContainerRuntime = "Docker"
```

Comment:
with Containerd going Stable in 1.20 should the default be ContainerD?

Reply:
Since we haven't published a lot of docs previously on how to configure and run with containerD I thought we can keep the defaults for these scripts using Docker for 1 more release.
What do you think?

Reply:
+1 for keeping docker for one more release. We can switch to containerd as default once people have used it more imo

Reply:
sgtm!